### PR TITLE
Default to 127.0.0.1:2375 on Windows

### PR DESCRIPTION
**Pull request by aanand**
Following the logic of the Docker client.

### CODE REVIEW

File: docker/utils/utils.py [hunk 1]
```diff
@@ -272,11 +272,15 @@ def parse_repository_tag(repo):
 # fd:// protocol unsupported (for obvious reasons)
 # Added support for http and https
 # Protocol translation: tcp -> http, unix -> http+unix
-def parse_host(addr):
+def parse_host(addr, platform):
```
Comment (shin-) [inline docker/utils/utils.py]: Couldn't we make `platform` optional?

Comment (aanand) [inline docker/utils/utils.py]: Ah, I didn't realise it was public API, so I guess we have to. Will fix.

Comment (aanand) [inline docker/utils/utils.py]: Done.

**Comment by mpetazzoni** [comment]
:+1:

**Pull request merged**